examples/large_models/Huggingface_pippy/Readme.md

```diff
@@ -6,23 +6,16 @@ PiPPy provides pipeline parallelism for serving large models that would not fit
 
 ## How to serve your large HuggingFace models with PiPPy in Torchserve?
 
-We use a Torchserve custom handler that inherits from base_pippy_handler to load the model and define our logic for preprocess, inference and post processing. This is basically very similar to your evaluation process.
+We use a Torchserve custom handler that inherits from base_pippy_handler to load the model and define our logic for preprocess, inference and post processing. This is basically very similar to your evaluation process. Following settings has been tested on g5.12xlarge EC2 instance which has 4xA10 GPUs.
 
 ### Step 1: Download model
```

Comment:
What about the pippy install steps? Will it be bundled as part of the base dependencies?

Reply:
Good catch, its in TS dev dependencies, will add to the example requirements.txt and call it out.